Dockerfile

```diff
@@ -210,7 +210,6 @@ RUN --mount=type=cache,sharing=locked,id=moby-containerd-aptlib,target=/var/lib/
         apt-get update && xx-apt-get install -y --no-install-recommends \
             gcc \
             libbtrfs-dev \
```

Comment:
this one should also be redundant now, as we're now on containerd 1.7, which no longer uses this;

- https://github.com/containerd/containerd/pull/7933